test/e2e/lib/auditlogs.go

```diff
@@ -25,6 +25,11 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+const (
```

Comment:
Are the two constant only used by the auditlogs test? If yes maybe it would be better to move them out of this package, or rename them to be less confusing?

Reply:
These two constants are used both in CloudAuditLogsSourceWithTestImpl of test/e2e/test_auditlogs.go and BrokerEventTransformationTestWithAuditLogsSourceHelper of test/e2e/lib/helpers/broker_event_transformation_test_with_source_helper.go. I moved it here to avoid cycle import. I will rename them to be less confusing.